service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -114,6 +114,9 @@ public class EhrExtractStatusService {
     private static final String CONTENT_TYPE_PLACEHOLDER = "CONTENT_TYPE_PLACEHOLDER_ID=";
     private static final String FILENAME_TYPE_PLACEHOLDER = "FILENAME_PLACEHOLDER_ID=";
     private static final String ACKS_SET = ACK_HISTORY + DOT + ACKS;
+    public static final int LIMIT_OF_DAYS_SINCE_EHR_EXTRACT_WAS_SENT = 8;
```

Comment:
Something sits funny with me about this variable name! I'm not quite sure what it is 🤔

Maybe something like:

* `MAX_DAYS_SINCE_EHR_EXTRACT_SENT`
* `EHR_EXTRACT_SENT_DAYS_LIMIT`
* `EHR_EXTRACT_EXPIRATION_DAYS`
* `EHR_EXTRACT_SENT_DAY_THRESHOLD`
* `EHR_EXTRACT_MAX_ALLOWED_DAYS`